fuse_constraints/include/fuse_constraints/absolute_pose_2d_stamped_constraint.h

```diff
@@ -67,16 +68,25 @@ class AbsolutePose2DStampedConstraint : public fuse_core::Constraint
   /**
    * @brief Create a constraint using a measurement/prior of the 2D pose
    *
-   * @param[in] position    The variable representing the position components of the pose
-   * @param[in] orientation The variable representing the orientation components of the pose
-   * @param[in] mean        The measured/prior pose as a vector (3x1 vector: x, y, yaw)
-   * @param[in] covariance  The measurement/prior covariance (3x3 matrix: x, y, yaw)
+   * Note that, when measuring subset of dimensions, empty axis vectors are permitted. This signifies, e.g., that you
+   * don't want to measure any of the quantities in that variable.
+   *
+   * @param[in] position        The variable representing the position components of the pose
+   * @param[in] orientation     The variable representing the orientation components of the pose
+   * @param[in] mean            The measured/prior pose as a vector (max 3x1 vector: x, y, yaw)
```

Comment:
mean/covariance are no longer strictly 3x1/3x3

Reply:
Yeah, that's why I changed the comment to "max 3x1 vector." Sounds like I should call that out more explicitly?

Reply:
No. I just somehow missed the "max". Ignore me.